languagetool-core/src/main/resources/org/languagetool/resource/spelling_global.txt

```diff
@@ -7624,3 +7624,4 @@ Fiat Scudo
 Citroën Jumpy
 Citroën C3
 Citroën C4
+Di Caprio
```

Comment:
His name is "DiCaprio" without a space (https://fr.wikipedia.org/wiki/Leonardo_DiCaprio)

Reply:
Oh, of course! I'll change to DiCaprio then (Leonardi DiCaprio already exists, but when using only the family name, the speller complains, as expected).

Reply:
Just a note here to say that a rule for that is created [here](https://github.com/languagetool-org/languagetool/pull/9420)